tests/cuda/test_cuda_containers.cpp

```diff
@@ -11,7 +11,6 @@
 #include "vecmem/allocators/allocator.hpp"
 #include "vecmem/containers/vector.hpp"
 #include "vecmem/memory/memory_manager.hpp"
```

Comment:
I think you could remove the "memory_manager.hpp" include as well. We can't remove that header until **all** of the "old style" memory management is removed, but at least this test should not include it anymore.

Reply:
Good one, fixed!